Kickstarter-iOS/Views/Cells/BackerDashboardEmptyStateCell.swift

```diff
@@ -19,17 +26,51 @@ internal final class BackerDashboardEmptyStateCell: UITableViewCell, ValueCell {
         .~ UIImage(named: "icon--eye",
                    in: .framework,
                    compatibleWith: nil)
+      self.filledHeartIconImageView.isHidden = true
     case .saved:
+      let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(animateToFilledHeart))
+      self.addGestureRecognizer(tapRecognizer)
       self.messageLabel.text = Strings.Tap_the_heart_on_a_project_to_get_notified()
       self.titleLabel.text = Strings.No_saved_projects()
-      _ = self.iconImageView
-        |> UIImageView.lens.image
-        .~ UIImage(named: "icon--heart-outline",
-                   in: .framework,
-                   compatibleWith: nil)
+      animateToFilledHeart()
     }
   }
 
+  @objc internal func animateToFilledHeart() {
+    UIView.animate(
+      withDuration: self.duration,
+      delay: 1.0,
+      usingSpringWithDamping: 1.0,
+      initialSpringVelocity: 0.0,
+      options: .curveEaseInOut,
+      animations: {
```

Comment:
Here, just for safety, could you make self inside the block optional? The line would become ` animations: { [weak self] in ...`